README.md

```diff
@@ -3379,7 +3379,167 @@ Response:
+The following query parameters are supported:
+
+| Parameter |                                        Description                                        | Required |
+|:---------:|:-----------------------------------------------------------------------------------------:|:--------:|
+| workflowA | Workflow ID of the first call                                                             | yes      |
+| callA     | Fully qualified name of the first call. **Including workflow name**. (see example below)  | yes      |
+| indexA    | Shard index of the first call                                                             | no       |
+| workflowB | Workflow ID of the second call                                                            | yes      |
+| callB     | Fully qualified name of the second call. **Including workflow name**. (see example below) | yes      |
+| indexB    | Shard index of the second call                                                            | no       |
+
+
+cURL:
+
+```
+$ curl "http://localhost:8000/api/workflows/v1/callcaching/diff?workflowA=85174842-4a44-4355-a3a9-3a711ce556f1&callA=wf_hello.hello&workflowB=7479f8a8-efa4-46e4-af0d-802addc66e5d&callB=wf_hello.hello"
+```
+
+HTTPie:
+
+```
+$ http "http://localhost:8000/api/workflows/v1/callcaching/diff?workflowA=85174842-4a44-4355-a3a9-3a711ce556f1&callA=wf_hello.hello&workflowB=7479f8a8-efa4-46e4-af0d-802addc66e5d&callB=wf_hello.hello"
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Content-Length: 1274
+Content-Type: application/json; charset=UTF-8
+Date: Tue, 06 Jun 2017 16:44:33 GMT
+Server: spray-can/1.3.3
+
+{
+  "callA": {
+    "workflowId": "85174842-4a44-4355-a3a9-3a711ce556f1",
+    "callFqn": "wf_hello.hello",
+    "jobIndex": -1,
+    "allowResultReuse": true
+  },
+  "callB": {
+    "workflowId": "7479f8a8-efa4-46e4-af0d-802addc66e5d",
+    "callFqn": "wf_hello.hello",
+    "jobIndex": -1,
+    "allowResultReuse": true
+  },
+  "hashDifferential": [
+    {
+      "command template": {
+        "85174842-4a44-4355-a3a9-3a711ce556f1:wf_hello.hello:-1": "4EAADE3CD5D558C5A6CFA4FD101A1486",
+        "7479f8a8-efa4-46e4-af0d-802addc66e5d:wf_hello.hello:-1": "3C7A0CA3D7A863A486DBF3F7005D4C95"
+      }
+    },
+    {
+      "input count": {
+        "85174842-4a44-4355-a3a9-3a711ce556f1:wf_hello.hello:-1": "C4CA4238A0B923820DCC509A6F75849B",
+        "7479f8a8-efa4-46e4-af0d-802addc66e5d:wf_hello.hello:-1": "C81E728D9D4C2F636F067F89CC14862C"
+      }
+    },
+    {
+      "input: String addressee": {
+        "85174842-4a44-4355-a3a9-3a711ce556f1:wf_hello.hello:-1": "D4CC65CB9B5F22D8A762532CED87FE8D",
+        "7479f8a8-efa4-46e4-af0d-802addc66e5d:wf_hello.hello:-1": "7235E005510D99CB4D5988B21AC97B6D"
+      }
+    },
+    {
+      "input: String addressee2": {
+        "85174842-4a44-4355-a3a9-3a711ce556f1:wf_hello.hello:-1": "116C7E36B4AE3EAFD07FA4C536CE092F",
+        "7479f8a8-efa4-46e4-af0d-802addc66e5d:wf_hello.hello:-1": null
```

Comment:
Yeah, kind of... I guess my complaint is that it's still a magic value, just not a humanly readable one... but I'm happy for you to leave it, especially since it **is** excellently documented :)